### DISCUSSION TITLE
Spinner is always spinning

### DISCUSSION THREAD

**Discussion by rabol** [wireui/wireui#844] [Q&A]
Hi

I must be doing something wrong, but cannot figure out what:

```
<x-button id="submit" primary wire:click="askConfirmDocument" spinner="submit" class="mt-4">{{ __('Approve') }}</x-button>
```

The above shows the spinner all the time after the page is loaded - why ?

**Comment by Uhasith** [comment]
Try like this. the Spinner prop should be same as wire:click method name i guess

```<x-button id="submit" primary wire:click="askConfirmDocument" spinner="askConfirmDocument" class="mt-4">{{ __('Approve') }}</x-button>```
